docs/advanced/plugins/index.md

```diff
@@ -22,6 +22,8 @@ redirect_from:
   </p>
 </blockquote>
 
+You can also find official plugins at https://github.com/babel-plugins and other community plugins by searching [npm](https://www.npmjs.com/search?q=babel-plugin-).
```

Comment:
Sorry I meant from the website (I just edited the file on the website which made the `patch-1` branch)